subsys/dsp/Kconfig

```
@@ -1,6 +1,15 @@
 # Copyright (c) 2022 Google LLC
 # SPDX-License-Identifier: Apache-2.0
 
+menuconfig DSP
+	bool "DSP subsystem"
+	help
+	  Include the (Digital Signal Processing) DSP subsystem as a part of the
+	  Zephyr build. This enables applications include and calling into the
+	  <zephyr/dsp/dsp.h> header.
```

```suggestion
	  Include the DSP (Digital Signal Processing) subsystem as a part of the
	  Zephyr build. This enables applications to include and call into the
	  <zephyr/dsp/dsp.h> header.
```

Super nit, sorry (😬), but since this /is/ public-facing and ends up in documentation, makes more sense to define the acronym after using it, and small suggestion for clarity.

---

Done